deadspace/code/necromorph/necromorphs/subtypes/brute.dm

```diff
@@ -34,6 +34,8 @@
 					break
 				destination = temp
 			if(destination != mob_target.loc)
+				playsound(mob_target.loc, "punch", 25, 1)
+				mob_target.apply_damage(24, BRUTE)
```

Comment:
This causes it to only play if the target would be flung, which is undesirable. You want to move this to nearly the top of the proc.